documentation/operators/src/changelog.md

```diff
@@ -2,6 +2,228 @@
+~~~
+
+- [Use `serde` from workspace](https://github.com/nymtech/nym/pull/4833): cargo autoinherit for `serde` - cargo autoinherit for `bs58` and `vergen` in `cosmwasm-smart-contracts` 
+
+- [Gateway database modifications for different modes](https://github.com/nymtech/nym/pull/4868): As gateway clients will not be solely from the mixnet, we need to split the table that handles shared keys from the client ids that are referenced from other tables. That way, the bandwidth table can be shared between different client types (entry mixnet, entry gateway, exit gateway), using the same `client_id` referencing.
+
+- [Remove the push trigger for `ci-nym-wallet-rust`](https://github.com/nymtech/nym/pull/4869)
+
+- [Chore: remove queued migration for adding explicit admin](https://github.com/nymtech/nym/pull/4871)
+
+- [Allow clients to send stateless gateway requests without prior registration](https://github.com/nymtech/nym/pull/4873): in order to make changes to the registration/authentication procedure we needed a way of extracting protocol information before undergoing the handshake. 
+
+- [Fix sql `serde` with `enum`](https://github.com/nymtech/nym/pull/4875)
+
+- [Few fixes to NNM pre deploy](https://github.com/nymtech/nym/pull/4883)
+
+- [Feature/updated gateway registration](https://github.com/nymtech/nym/pull/4885): This PR introduces support for aes256-gcm-siv shared keys between clients and gateways.
+    - Those changes should be fully backwards compatible. if they're not, there's a bug.
+~~~admonish example collapsible=true title='Testing steps performed'
+- For the following combinations I inited the client, ran the client, stopped the client, and ran the client again:
+- Fresh client on new binary && gateway on old binary
+- Fresh client on old binary && gateway on new binary
+- Fresh client on new binary && gateway new binary 
+- Existing old client on old binary && new gateway 
+~~~
+ 
+- [Build and Push CI](https://github.com/nymtech/nym/pull/4887)
+
+- [Entry wireguard tickets](https://github.com/nymtech/nym/pull/4888): Note: The behaviour of the nodes and vpn client (as a test) has not changed, it still works as it used to. Obtaining ticketbooks also is unchanged
+
+- [Update `nym-vpn` metapackage and replace `nymvpn-x` with `nym-vpn-app`](https://github.com/nymtech/nym/pull/4889): Change dependency from `nymvpn-x` to `nym-vpn-app` to reflect the new package name of the tauri client
+
+- [Update network monitor entry point](https://github.com/nymtech/nym/pull/4893)
+
+- [Remove clippy github PR annotations](https://github.com/nymtech/nym/pull/4896): It eats up CI resources and time to run the clippy annotation checks that likely no one uses anyway. We keep the clippy checks of course.
+
+- [Fix clippy for beta toolchain](https://github.com/nymtech/nym/pull/4897): 
+
+- [Update cargo deny](https://github.com/nymtech/nym/pull/4901): Update to use latest `cargo-deny`.
+  - Regenerate `deny.toml`
```

Comment:
Should this stuff be in a an `~~~admonish example collapsible=true title='Try yourself'`?